community/README.md

```diff
@@ -4,5 +4,38 @@ title: Community
 permalink: /docs/community
 ---
 
-TODO
+Vuforia Spatial Toolbox is an open source project, licensed under MPL 2.0, and we encourage you
+to join our community and contribute to the project in whatever way you can. Anyone is welcome
+and encouraged to be a part of our community, whether you're a student looking to learn about
+Augmented Reality and Spatial Computing, a hobbyist looking to experiment with some DIY projects,
+an innovation engineer looking to use this in your industry, or a researcher looking to push the
+boundaries of this platform even further.
 
+## How to Participate
+
+### Forum
+
+The main hub of our community is the [Spatial Toolbox Forum](http://forum.spatialtoolbox.vuforia.com).
+Introduce yourself, ask questions, share your knowledge, discuss new ideas, and inspire others by
+showing what you've built.
+
+### GitHub
+
+All of the code is hosted on the [PTC Reality Lab GitHub](https://github.com/ptcrealitylab) in a
+handful of repositories. Clone these to compile customized versions of the software that you can
+modify with new features. Feel free to fork these and make your own changes, which you can share
+back to the community.
+
```

Comment:
It would be nice to have a link to https://github.com/firstcontributions/first-contributions/blob/master/README.md or another GitHub fork->clone->PR tutorial here

Reply:
Oh, good idea! I will update with this